php/common/control.php

```diff
@@ -113,21 +115,20 @@
                 continue;
             }
             // пропустим исключённые из регулировки подразделы
-            $subControlPeers = isset($cfg['subsections'][$forumID]) ? (int)$cfg['subsections'][$forumID]['control_peers'] : 0;
+            $subControlPeers = isset($cfg['subsections'][$forumID]) ? $cfg['subsections'][$forumID]['control_peers'] : "";
```

Comment:
A bit late now for review comments, but how correct is it to have `""` here when `get_control_peers` expects all arguments to be integers?

Probably a value, like `-2`, that does not trigger `continue` just below can fit here?

Reply:
Good point, but default values of `clientControlPeers` and `subControlPeers` from settings - is empty string, so if cast it to int, it will be 0, and calculations will be wrong.
Probably, best way is check like this ` != ""`, cos `-1` values a filtered before `get_control_peers` call.